app/controllers/episode_media_controller.rb

```diff
@@ -32,6 +32,7 @@ def update
       if @episode.save
         @episode.uncut&.slice_contents!
         @episode.copy_media
+        @episode.publish!
```

Comment:
we need to also update integrations like megaphone when the media controller has updates

Reply:
Currently, we copy the media and then later (after Porter callbacks, if any) we trigger `ep.publish!`.

Does that not catch this case?  Or you just don't want to wait for Porter to kick off a publishing pipeline?

Reply:
No, it doesn't catch when there is a draft episode, and the number of segments is changes, for example.